lib/generators/grnds/onetime/install/install_generator.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails/generators'
+
+module Grnds
+  module Onetime
+    class InstallGenerator < Rails::Generators::Base
+      def add_migration
```

Comment:
I had it in mind to not have to rely on migrations to create the table, but instead check if the table exists first and use the rake task to execute sql to create the table. Internally the Migrator does the same thing.